app/views/content/covid_19_guidance.md

```diff
@@ -0,0 +1,54 @@
+* To give you more time to make decisions, we won’t reject or decline any offers automatically until 20 April 2020. Below we outline what this means for you and your candidates. We’ll keep you updated if anything else changes. 
+
+## Giving you more time before applications are rejected by default (RBD)
+
+### Applications that have been submitted already
+
+You have until 23:59 on 20 April to make decisions on applications which would normally be automatically rejected on or before 20 April.
+
+For applications which have a RBD date after 20 April, you’ll also have more time to make a decision. We’ll add an extra 4 weeks to the response deadlines.
+
+### Applications that are submitted between now and 20 April
+
+If applications come in between now and 20 April, you have until 23:59 on 15 June to respond. No applications in this category will be automatically rejected until then.
+
+### How to keep track of your new RBD dates
+
+You can see your new RBD dates in [Manage teacher training applications](/provider/applications).
+
+### Keeping candidates up to date
+
+We’ll let candidates know that it might take you longer to respond to their application, and tell them your new RBD date. 
+
+## Giving candidates more time to make decisions on offers
+
+### Hearing back from candidates might take longer
+
+We’re pausing all automatic decline by default (DBD) dates until 20 April. This means that there may be a delay hearing back from any candidates who would normally have to respond to an offer on or before 20 April. We’re giving everyone in this category until 23:59 on 20 April to respond. 
```

Comment:
Is this section accurate? I thought that the clock would just be paused until 20 April some candidates will be able to respond later.

Reply:
not sure I understand what you mean @stevehook ?